src/EnergyPlus/DataMoistureBalanceEMPD.hh

```diff
@@ -32,6 +32,15 @@ namespace DataMoistureBalanceEMPD {
 	extern Array1D< Real64 > MoistEMPDNew; // Moisture level at interior surfaces at current interation
```

Comment:
@jdwoods21 Which of these do we not need anymore? From what I can see, I should be able to get rid of:

- `MoistEMPDOld`
- `MoistEMPDInt`
- `MoistEMPDNew`
- `MoistEMPDFlux`

Is that right?